packages/airnode-node/src/evm/workers.test.ts

```diff
@@ -24,31 +22,33 @@ const providerErrorForWorker = {
 workers.forEach((workerType) => {
   describe(`${workerType} worker`, () => {
     fixtures.setEnvVariables({ AIRNODE_WALLET_PRIVATE_KEY: fixtures.getAirnodeWalletPrivateKey() });
+    const state = fixtures.buildEVMProviderSponsorState();
+    const mockInvokeCommandInput = {
+      FunctionName: serverlessFunctionName,
+      Payload: new TextEncoder().encode(JSON.stringify({ state, functionName: functionNames[workerType] })),
+    };
+
+    beforeEach(() => {
+      mockLambdaClient.reset();
```

Comment:
Doesn't it reset on it's own?

Reply:
I'm not sure, I was following the documentation. It won't hurt to keep it.